apps/labs/posts/nebari-typer-cli.md

```diff
@@ -0,0 +1,162 @@
+</p>
+
+We can see that the `nebari init` command has a lot of available options. If we are using Nebari first time and we don't know exactly what other each of the options represent, then running `nebari init --guided-init` will guide us through a simple questionnaire which provides helpful information ,and links to additional resources, to assist with a number of important decisions.
+
+```bash
+  `nebari init --guided-init`
+```
+
+<p align="center">
+    <img
+     alt="Output of the nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi.png" 
+    />
+    <img
+     alt="Output of the nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi1.png" 
+     />
+    <img
+     alt="Output of the nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi2.png" 
+     />
+    <i>Output of nebari init guided-init command</i>
+</p>
+
+As we can see it's more like a survey that asks end-users to fill out the details for generating the `nebari-config.yaml` file. To create this questionnaire, We have used the [questionary library](https://github.com/tmbo/questionary).
+
+After generating the `nebari-config.yaml`, it's time to validate and render it respectively. First we will see the `help` command output for the validate to see all the options and requirements available for it.
+
+```bash
+  `nebari validate --help && nebari render --help`
+```
+
+<p align="center">
+    <img
+     alt="Output of the nebari validate and render help"
+     src="/posts/asmi-blogposts/nebari-vrh.png" />
+    <i>Output of the nebari validate and render help command</i>
+</p>
+
+As we can see we have to pass the `-c/--config` flag for validating the `nebari-config.yaml` file and we have to do the same for rendering the infrastructure using the render command.
+
+```bash
+  `nebari validate -c qhub-config.yaml && nebari render -c qhub-config.yaml`
+```
+
+<p align="center">
+    <img
+     alt="Output of the nebari validate and neabri render"
+     src="/posts/asmi-blogposts/nebari-rv.png" />
+    <i>Output of the nebari validate and render command</i>
+</p>
+
+The render command renders all of the Terraform scripts used for the actual deployment. These Terraform scripts are broken into stages and are stored in directory called `stages`.
+
+**_NOTE:_** It's not necessary to run the `validate` and `render` commands separately as both commands are run under the hood by the deploy command.
+
+Now we will see the output of the `nebari deploy` help command. The deploy command creates the provisioned resources that Nebari requires and deploys your project. Nebari also has a destroy command that works the same way the deploy works but instead of creating the provisioned resources it destroys it. The destroy command now requires confirmation, an important new feature. Previously it didn't ask for the confirmation and the cluster would start being destroyed right after executing it.
```

Comment:
```suggestion
Now we will see the output of the `nebari deploy` help command. The deploy command creates the necessary resources Nebari needs. Nebari also has a destroy command that works in a similar manner but instead of creating resources, it destroys them. The destroy command now requires confirmation, an important new feature. Previously it didn't ask for the confirmation and the cluster would start being destroyed right after executing it.
```